src/Database/Schema/TableSchemaAwareInterface.php

```diff
@@ -24,15 +24,15 @@ interface TableSchemaAwareInterface
     /**
      * Get and set the schema for this fixture.
      *
-     * @return \Cake\Database\Schema\TableSchemaInterface|null
+     * @return \Cake\Database\Schema\TableSchemaInterface&\Cake\Database\Schema\SqlGeneratorInterface
      */
-    public function getTableSchema(): ?TableSchemaInterface;
+    public function getTableSchema();
 
     /**
      * Get and set the schema for this fixture.
      *
-     * @param \Cake\Database\Schema\TableSchemaInterface $schema The table to set.
+     * @param \Cake\Database\Schema\TableSchemaInterface$schema $schema The table to set.
```

Comment:
This was messed up by `stickler`. Problem lies in our `FunctionCommentSniff` which doesn't account to `&` in param type :disappointed:.

Reply:
Opened cakephp/cakephp-codesniffer#252 for this.